lib/utilities/js/is-even.ts

```diff
@@ -0,0 +1,5 @@
+function isEven(num: number): boolean {
+  return num % 2 === 0;
```

Comment:
Try to use **Math.abs()** to support negative integers and **Number.isInteger()** to avoid errors in case of numbers other that integers. There is no reason to support floats or other type of numbers.
A solution could be something like code below.
```js
function isEven(num) {
  if (Number.isInteger(num)) {
    return Math.abs(num) % 2 === 0;
  }
}
```